frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -16,14 +19,31 @@ export class OfferDetailComponent implements OnInit {
   public offer$: Observable<Offer>;
   public djangoRoot = environment.djangoRoot;
   public getJoinViewUrl = this.offersService.getJoinViewUrl;
+  isUserOrgMember$: Observable<boolean>;
+  user$: Observable<User | null>;
 
   constructor(
     private activatedRoute: ActivatedRoute,
+    private authService: AuthService,
     private offersService: OffersService,
   ) { }
 
   ngOnInit() {
-    this.offer$ = this.activatedRoute.params
-    .switchMap(params => this.offersService.getOffer(params.offerId));
+    this.user$ = this.authService.user$
+
+    this.offer$ = this.offersService.offer$
+
+    this.activatedRoute.params
+    .switchMap(params => this.offersService.getOffer(params.offerId))
+    .subscribe();
+  
+    this.isUserOrgMember$ = this.offer$
+     .combineLatest(this.user$, (offer, user): boolean => {
+       if (offer === null || user === null) {
```

Comment:
if (!offer || !user)

Reply:
You can also rewrite the whole if-else block. 'if' with 'positive' condition and drop 'else' - just follow it with a return false, like this:

if (offer && user) {
   // return filtered organizations
} 
return false;